src/tests/api.rs

```diff
@@ -212,5 +212,6 @@ fn exercise_full_api() {
             let _: String = nfr.name;
             let _: u32 = nfr.ix;
         }
+        nodes::NodeValue::MultilineBlockQuote(_) => {}
```

Comment:
Let's add detail of the fields (per above), so that we catch any API-breaking changes to the multiline block quote.